### ISSUE TITLE
Scheduler "noop" is not valid for Multiqueue

### ISSUE THREAD

**Issue by pierre-labastie** [gentoo/eudev#146]
eudev-3.2.2, linux-4.11.7, with [LFS](http://www.linuxfromscratch.org). Part of the init script:
```
      # Start the udev daemon to continually watch for, and act on,
      # uevents
      /sbin/udevd --daemon

      # Now traverse /sys in order to "coldplug" devices that have
      # already been discovered
      /sbin/udevadm trigger --action=add    --type=subsystems
      /sbin/udevadm trigger --action=add    --type=devices
      /sbin/udevadm trigger --action=change --type=devices
```
I get:
```
Jun 26 20:30:11 turboli kernel: [    3.100376] elevator: switch to noop failed
Jun 26 20:30:11 turboli kernel: [    3.100758] elevator: switch to noop failed
Jun 26 20:30:11 turboli kernel: [    3.101208] elevator: switch to noop failed
Jun 26 20:30:11 turboli kernel: [    3.101593] elevator: switch to noop failed
```
Using udevadm test, it appears the this occurs when "adding" /sys/block/nbd? devices.
Reason: the only possible values for ATTR{queue/scheduler} are *mq-deadline* and *none*, but 
**/lib/udev/rules.d/60-block.rules** has:
```
# set noop on solid state drives
SUBSYSTEM=="block", ACTION=="add", ATTR{queue/rotational}=="0", ATTR{queue/scheduler}="noop"
```
I am not sure how this should be corrected (on my system, I just removed that line, since I have no SSD).

**Comment by blueness** [comment]
yep that line has to go.  someone asked me to add it, but its wrong.

**Comment by blueness** [comment]
This is fixed in 47367bc4df6f05a49cf4b5b0209153d77a9ade83
